Multisig/UI/Safe Management/SafeInfoView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SafeInfoView.swift
+//  Multisig
+//
+//  Created by Moaaz on 4/23/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+import CoreData
+
+struct SafeInfoView: View {
+    @FetchRequest(fetchRequest: AppSettings.settings()) var appSettings: FetchedResults<AppSettings>
+    @Environment(\.managedObjectContext) var context: NSManagedObjectContext
+    
+    var body: some View {
+        var safe: Safe?
```

Comment:
Can we provide here Safe object in the constructor, as it is already known when we call this view (I guess so, not sure)?